src/Cardano/CoinSelection/LargestFirst.hs

```diff
@@ -37,12 +38,6 @@ import qualified Data.Map.Strict as Map
 
 -- | Generate a coin selection according to the __largest first__ algorithm.
 --
--- === Summary
---
--- For the given /output list/ and /initial UTxO set/, this algorithm generates
--- a /coin selection/ that is capable of paying for all of the outputs, and a
--- /remaining UTxO set/ from which all spent values have been removed.
---
```

Comment:
This section is no longer necessary, as we have equivalent documentation in the `CoinSelectionAlgorithm` type.